shotover-proxy/example-configs-docker/redis-cluster/docker-compose.yml

```diff
@@ -0,0 +1,100 @@
+version: '2'
```

Comment:
What is example-configs-docker/redis-cluster for?

Reply:
For the "Unaware client" Redis example, instead of running the Shotover binary yourself, I added the Docker image to the `docker-compose.yml` to be more consistent with the other examples.

I copied the config files for `example-configs/redis-cluster` to the new folder, changed them and left the original ones in the `example-configs` dir because they are used by some tests.

Reply:
For the "Unaware client" Redis example, instead of running the Shotover binary yourself, I added the Docker image to the `docker-compose.yml` to be more consistent with the other examples.

I copied the config files for `example-configs/redis-cluster` to the new folder, changed them and left the original ones in the `example-configs` dir because they are used by some tests.

Reply:
Thanks
I'll try and review this all in place.
For next time though, this PR has a lot of stuff going on now and some of it could have been in follow up PRs.